README.md

```diff
@@ -1289,6 +1289,27 @@ Other Style Guides
 
   - [15.4](#15.4) <a name='15.4'></a> For more information see [Truth Equality and JavaScript](http://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/#more-2108) by Angus Croll.
 
+  - [15.5](#15.5) <a name='15.5'></a> Ternaries should not be nested and generally be single line expressions.
```

Comment:
I don't think there's anything wrong with multi-line ternary statements. 
```js
const someValue = this.hasSomeThing() 
    ? this.someThing(param)
    : this.otherThing(param);
```
Is clearer than
```js
const someValue = this.hasSomeThing() ? this.someThing(param) : this.otherThing(param);
```